task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -133,17 +165,23 @@ def __handle_deleted_pod_event(self, event: PodEvent) -> None:
                 terminal=True,
                 success=False,
                 timestamp=time.time(),
-                raw=event["raw_object"],
+                raw=raw_event,
                 task_config=task_metadata.task_config,
                 platform_type="killed"
             )
         )
 
     def __handle_modified_pod_event(self, event: PodEvent) -> None:
         pod = event["object"]
+        self.__update_modified_pod(pod=pod, event=event)
+
+    def __update_modified_pod(self, pod: V1Pod, event: Optional[PodEvent]) -> None:
+        """ Called during reconciliation and normal event handling """
         pod_name = pod.metadata.name
         task_metadata = self.task_metadata[pod_name]
 
+        raw_event = event['raw_object'] if event else ''  # TODO: when is this empty?
```

Comment:
Whoops, forgot to remove this TODO but I did have a question -- do you think it's worth crafting an event string here that indicates this came from reconciliation? 
I could not find where we use this in tron on the kubernetes side, but on the mesos side we strictly used it as a string in error reporting.

Reply:
`raw_object` is normally just a V1Pod in `dict` form at the moment for `task` events and probably not very useful for most people :p 

I think that if we want to report on reconciliation events, we'd maybe want to send a `control` events, but probably only if we want to take action on the fact that we reconciled/didn't reconcile something

imo, maybe we just want to send `None` in `raw` for reconciled events? that way, the data structures are a little more clear - Optional[Dict] instead of Union[Dict, str]

Reply:
Cool, I have just left these `None` for reconciled events.